finish/inventory/pom.xml

```diff
@@ -151,15 +151,11 @@
                     <execution>
                         <id>package-server</id>
                         <configuration>
-                            <packageFile>${project.build.directory}/defaultServer.zip</packageFile>
```

Comment:
These pom changes need to be done for `job` and `gateway` as well to stay consistent with these changes. @justineechen @evelinec